### PR TITLE
Boot reqProc using a new script

### PR DESCRIPTION
**Pull request by shrivara**
https://github.com/Shippable/node/issues/112

### CODE REVIEW

File: _boot.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+set -o pipefail
+
+# Main directories
+readonly NODE_SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly SHIPPABLE_DIR="/etc/shippable"
+
+# Node Sub-directories
+readonly SCRIPTS_DIR="$NODE_SCRIPTS_DIR/initScripts"
```
Comment (ric03uec) [inline _boot.sh]: wouldn't it be better to move all the node-specific env related stuff in a different file and source it

File: _boot.sh [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+readonly SWAP_FILE_PATH="/root/.__sh_swap__"
+
+# Misc
+readonly BOOT_WAIT_TIME=10
+
+# Source libraries
+source "$LIB_DIR/logger.sh"
+source "$LIB_DIR/headers.sh"
+
+check_input() {
```
Comment (ric03uec) [inline _boot.sh]: something like this should go in the `lib/_helpers`

File: _boot.sh [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  source $NODE_INIT_SCRIPT
+
+  local expected_envs=(
+    'REQPROC_CONTAINER_NAME_PATTERN'
+    'REQPROC_OPTS'
+    'REQPROC_MOUNTS'
+    'REQPROC_ENVS'
+  )
+
+  for env in "${expected_envs[@]}"
```
Comment (ric03uec) [inline _boot.sh]: add a common function in helpers to do this,

File: initScripts/x86_64__Ubuntu__16.04__Docker__17.06.sh [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+    echo "NTP already installed, skipping."
+  else
+    echo "Installing NTP"
+    exec_cmd "sudo apt-get install -y ntp"
+    exec_cmd "sudo service ntp restart"
+  fi
+}
+
+setup_mounts() {
+  export BASE_DIR="$SHIPPABLE_DIR/$(cat /proc/sys/kernel/random/uuid)"
```
Comment (ric03uec) [inline initScripts/x86_64__Ubuntu__16.04__Docker__17.06.sh]: should be at the top of this file,

File: _boot.sh (hunk 1 above)
Comment (shrivara) [reply _boot.sh]: I'm not sure where I can move this. Lib?

**Pull request merged**